codeforamerica/vita-min
# Linux development

## Link to pivotal/JIRA issue
https://github.com/newjersey/affordability-pm/issues/26
## Is PM acceptance required? (delete one)
No
## What was done?
Add an optional docker compose setup without disrupting local install
## How to test?
To make sure that local development is not disrupted by this PR, pull down this PR and run local development. Verify that the tests run and the database connection works.

One outstanding question that's marked todo: what should the default username and password be for local install postgres?

To try docker setup, first stop all running servers for this project, as the same ports are used in both. Then, `docker compose up`. Follow the commands added to the readme to run tests in docker.
## Screenshots (for visual changes)
There should be no visual changes
